tests/unit/copydb.sh

```diff
@@ -45,7 +43,20 @@ do
     t=`basename $f .sql`
     r=/tmp/results/${t}.out
     e=./expected/${t}.out
-    psql -d "${PGCOPYDB_TARGET_PGURI}" ${pgopts} --file ./sql/$t.sql &> $r
+    psql -d "${PGCOPYDB_TARGET_PGURI}" ${pgopts} --file $f &> $r
     test -f $e || cat $r
     diff $e $r || exit 1
 done
+
+
+for f in ./script/*.sh
+do
+    t=`basename $f .sh`
+    r=/tmp/results/${t}.out
+    e=./expected/${t}.out
+    bash $f $r
```

Comment:
I think it would be better that the test script outputs to stdout and we capture that here. In other words:

``` diff
- bash $f $r
+ bash "$f" > "$r"

Reply:
You can still use `2>&1` at the end of the `pgcopydb` call and still get the logs in the outfile, which I think was nicer anyways, because it serves as documentation too...